### PR TITLE
update contributing guide

### PR DESCRIPTION
**Pull request by IreTheKID**

### CODE REVIEW

**Comment by leon332157** [comment]
We will probably update this later once we have a mock DB instead of actually needing to create one on the altas.

**Comment by IreTheKID** [comment]
@leon332157 alright then, should I close this?

**Comment by leon332157** [comment]
Leave it open. We are gonna add based on this.

**Comment by IreTheKID** [comment]
Ok sure

File: README.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
- [![Run on Repl.it](https://repl.it/badge/github/mat-1/ReplDex)](https://repl.it/github/mat-1/ReplDex)
+[![Run on Repl.it](https://repl.it/badge/github/mat-1/ReplDex)](https://repl.it/github/mat-1/ReplDex)
+
+# Repldex
+Official encyclopedia of user created entries for the Replit Discord Server. Contains the Discord bot source code and `aiohttp` web server (API and frontend).
```
Comment (mat-1) [inline README.md:4]: I don't think Repldex is official lol

Comment (IreTheKID) [reply README.md:4]: doesn't have to be tbh =P

**Comment by IreTheKID** [comment]
@mat-1 merge when

**Comment by mat-1** [comment]
> 
> 
> @mat-1 merge when

When you change official to unofficial so Repl.it staff don't get mad at us lol

**Comment by IreTheKID** [comment]
lol fine

**Comment by IreTheKID** [comment]
@mat-1 done =P

**Review by mat-1 — APPROVED** [review]

**Pull request merged**